ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressInput.service.ts

```diff
@@ -365,8 +365,8 @@ export class AddressInputService implements OnDestroy {
         const newFieldValue = address[propName];
         if (formControl && newFieldValue !== formControl.value) {
           formControl.patchValue(newFieldValue, { emitEvent: emitValueChange });
-          markFieldsAsTouched && formControl.markAsTouched();
         }
+        markFieldsAsTouched && formControl.markAsTouched();
```

Comment:
Found another issue that when we load a saved address (from profile or a temporary one) we would not highlight in the form missing values (e.g., the  NAME field was missing). To make sure formgroup validations are visible, we touch all the fields in the form when load an address.